### DISCUSSION TITLE
Upcoming features

### DISCUSSION THREAD

**Discussion by astrandb** [astrandb/miele#10] [Announcements]
Look here for device diagnostics feature. Needs Home Assistant v 2022.2 or later.
PR https://github.com/astrandb/miele/pull/8

**Comment by mundschenk-at** [comment]
While useful, that's not something an automation can react to, no? Attributes on a sensor would allow for automations to be written for "unsupported" device types (at least as an interim measure.

**Reply by astrandb** [reply]
- Correct, only for diagnostics & debugging.
- It will probably be quite quick release cycles for new device types as soon as the basics are sorted out. I would rather avoid interim solutions that lead to breaking changes in automations and scripts every time the ordinary support is implemented. I shall consider your suggestion when I dive into the code nex time. No promises on delivery time in open source development though.
